airflow/models/baseoperator.py

```diff
@@ -1371,6 +1371,11 @@ def is_smart_sensor_compatible(self):
         """Return if this operator can use smart service. Default False."""
         return False
 
+    @property
+    def inherits_from_dummy_operator(self):
+        """Used to determine if an Operator is inherited from DummyOperator"""
```

Comment:
```suggestion
        """Used to determine if an Operator is inherited from DummyOperator"""
        # This looks like `isinstance(self, DummyOperator) would work, but this also needs to cope when `self` is a Serialized instance of a DummyOperator or one of its sub-classes (which don't inherit from anything but BaseOperator).
```

But with line wrapping

Reply:
Pushed: https://github.com/apache/airflow/pull/12745/commits/b150a6a60be8a2cffd22069e797a48e30d41967f